rust/datafusion/src/variable/system.rs

```diff
@@ -0,0 +1,18 @@
+use crate::logicalplan::ScalarValue;
+use crate::error::Result;
+use crate::variable::VariableProvider;
+
+pub struct SystemVariable {}
+
+impl SystemVariable {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+impl VariableProvider for SystemVariable {
+    fn get_value(&self, variable_names: Vec<String>) -> Result<ScalarValue> {
+        let s = format!("{}-{}", "test".to_string(), variable_names.concat());
```

Comment:
why `test`?

Reply:
When the user USES, to create a new system and user_defined method, here is only for the user to understand.
The @@ variable is defined by the external system